src/Gui/NaviCube.cpp

```diff
@@ -176,12 +179,19 @@ class NaviCubeImplementation : public ParameterGrp::ObserverType {
 		TEX_TOP_LEFT_REAR,
 		TEX_TOP_REAR_RIGHT,
 		TEX_ARROW_NORTH,
+		TEX_ARROW_NORTH_PICK,
```

Comment:
Many thanks for the PR! Since FC 0.20 is in feature freeze, I cannot put in new features. (Others might be offended whose features have to wait.) Can you please omit everything that is not needed to fix the rendering issue?
For example you changed the appearance of the cube corner. This should be omitted.